UnityProject/Assets/Scripts/Tool/Multitool.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Multitool : MonoBehaviour, ICheckedInteractable<PositionalHandApply>
```

Comment:
This needs some interaction messages so that players (and nearby players) can understand what is happening when the tool is being used.